builtin/common/information_formspecs.lua

```diff
@@ -56,28 +56,27 @@ local function build_chatcommands_formspec(name, sel, copy)
 	local description = "For more information, click on any entry in the list.\n" ..
 		"Double-click to copy the entry to the chat history."
 
-	for i, data in ipairs(mod_cmds) do
+	for _, data in ipairs(mod_cmds) do
 		rows[#rows + 1] = COLOR_BLUE .. ",0," .. formspec_escape(data[1]) .. ","
-		for j, cmds in ipairs(data[2]) do
+		for _, cmds in ipairs(data[2]) do
 			local has_priv = check_player_privs(name, cmds[2].privs)
 			rows[#rows + 1] = ("%s,1,%s,%s"):format(
-				has_priv and COLOR_GREEN or COLOR_GRAY,
+					has_priv and COLOR_GREEN or COLOR_GRAY,
 				cmds[1], formspec_escape(cmds[2].params))
```

Comment:
I think here it should be only one indentation because it's not an if (or while, for etc.) condition.